libraries/esp8266/examples/BlinkPolledTimeout/BlinkPolledTimeout.ino

```diff
@@ -59,13 +58,11 @@ void setup() {
 
   pinMode(LED_BUILTIN, OUTPUT);     // Initialize the LED_BUILTIN pin as an output
 
-  using esp8266::polledTimeout::oneShotMs; //import the type to the local namespace
-
   //STEP1; turn the led ON
   ledOn();
 
   //STEP2: wait for ON timeout
-  oneShotMs timeoutOn(2000);
+  esp8266::polledTimeout::oneShotMs timeoutOn(2000);
```

Comment:
Why remove the above using statement and fully qualify here? The current code is better, because you can change just the using statement above to specify how to import the type, and do that with a 1-liner to cover both uses below.